src/backend/src/services/projects.services.ts

```diff
@@ -15,6 +15,7 @@ import {
 } from '../utils/projects.utils';
 import { descBulletConverter, wbsNumOf } from '../utils/utils';
 import { createDescriptionBulletChangesJson } from '../utils/work-packages.utils';
+import WorkPackagesService from './work-packages.services';
```

Comment:
ik it's cuz the other stuff is commented but lets delete this for now since the linter will pick it up in every pr